build.gradle

```diff
@@ -218,19 +214,15 @@ task copyToLibs(type: Copy) {
 }
 
 task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
-	outputs.files.setFrom(file("$buildDir/libs/pcgen-${version}.jar"))
```

Comment:
the pcgen jar is already included in the 'from' clause in the following line so there was no need to include the file twice. Also, if you are just trying to make sure that jar is part of the task output that happens automatically for you. Or is there perhaps a more nuanced reason for this line to be there that I didn't catch?